templates/hesabe_app/pay.html

```diff
@@ -15,14 +15,14 @@ <h5>Hesabe Payment gateway</h5>
 			<input type="text" hidden name="variable3" value = {{variable1}}>
 			<input type="text" hidden name="variable4" value = {{variable1}}>
 			<input type="text" hidden  name="variable5" value = {{variable1}}>
-			<input type="text" hidden  name="Orderid" value = {{order_id}}>
+			<input type="text" hidden  name="orderReferenceNumber" value = {{orderReferenceNumber}}>
 			
```

Comment:
need to have a provision to accept currencies too. You can check the parameter in the docs.

Reply:
That was a follow-up task, which was supposed to be assigned later to me. Will take care of it in later commits.